content/includes/footer.html

```diff
@@ -101,7 +101,9 @@ <h3>Get help from Veterans Crisis Line</h3>
 {% if buildtype === 'development' %}
   <script>
     window.dataLayer = [];
+    window.dataLayer.push({ environment: '{{ buildtype }}' });
```

Comment:
Is there any chance that top line should be `window.dataLayer = window.dataLayer || [];`?  I thought I read once that it's possible that it could have been initialized somewhere else with data already pushed to it.

Reply:
yeah that's true. we do it that way in the production snippet below for that reason.

for dev we were just lazier since dropped data didn't matter that much. i'll update it though as the consistency is a good idea.